File: Exercise6/Board.java
class Board {

    private Piece[][] board;

    public Board() {
        this.board = new Piece[8][8];
    }

    public Piece get(Position p) {
        return board[p.getX()][p.getY()];
    }

    public void set(Position p, Piece piece) {
        board[p.getX()][p.getY()] = piece;
    }
}

File: Board.cs
namespace Chess {

    internal class Board {

        private Piece[,] board;

        public Board() {
            this.board = new Piece[8, 8];
        }

        public Piece getPieceAt(Position pos) {
            return null;
        }

        public void setPieceAt(Piece piece, Position pos) {
            this.board[pos.X, pos.Y] = piece;
        }
    }
}
